src/components/auth/login/Login.vue

```diff
@@ -1,51 +1,76 @@
 <template>
-  <div class="login">
-    <div class="display-2 text--center my-4">{{ $t('auth.welcome') }}</div>
-    <form method="post" action="/auth/login" name="login">
-      <div class="form-group">
-        <div class="input-group">
-          <input type="text" id="email" required="required"/>
-          <label class="control-label" for="email">
-            {{ $t('auth.email') }}
-          </label>
-          <va-icon icon="bar"/>
-        </div>
-      </div>
-      <div class="form-group">
-        <div class="input-group">
-          <input type="password" id="password" required="required"/>
-          <label class="control-label" for="password">
-            {{ $t('auth.password') }}
-          </label>
-          <va-icon icon="bar"/>
-        </div>
-      </div>
-      <div
-        class="d-flex align--center justify--space-between">
-        <va-button type="submit">
-          {{ $t('auth.login') }}
-        </va-button>
-        <va-button flat :to="{ name: 'signup' }">
-          {{ $t('auth.createAccount') }}
-        </va-button>
-      </div>
-    </form>
-  </div>
+  <form class="login" @submit="onsubmit">
+    <div class="va-row mb-4">
+      <va-input
+        v-model="email"
+        type="email"
+        :label="$t('auth.email')"
+        :error="!!emailErrors.length"
+        :error-messages="emailErrors"
+      />
+    </div>
+    <div class="va-row mb-4">
+      <va-input
+        v-model="password"
+        type="password"
+        :label="$t('auth.password')"
+        :error="!!passwordErrors.length"
+        :error-messages="passwordErrors"
+      />
+    </div>
+    <div class="va-row align--center justify--space-between login__actions">
+      <va-checkbox v-model="keepLoggedIn" class="mb-0" :label="$t('auth.keep_logged_in')"/>
+      <router-link :to="{name: 'recover-password'}">{{$t('auth.recover_password')}}</router-link>
+    </div>
+    <div class="va-row justify--center">
+      <va-button type="submit" class="my-0">{{ $t('auth.login') }}</va-button>
+    </div>
+  </form>
 </template>
 
 <script>
 export default {
   name: 'login',
+  components: {},
+  data () {
+    return {
+      email: '',
+      password: '',
+      keepLoggedIn: false,
+      emailErrors: [],
+      passwordErrors: [],
+    }
+  },
+  methods: {
+    onsubmit () {
```

Comment:
I suggest you add a computed property to indicate form readiness.
{
  formReady () {
    return !this.emailErrors.length && !this.passwordErrors.length
  }
}

```
      if (!this.email) {
        this.emailErrors.push('Email is required')
      }

      if (!this.password) {
        this.passwordErrors.push('Password is required')
      }

      if (!this.formReady) {
        return
      }

      this.$router.push({ name: 'dashboard' })
```